src/Engine/Entity/Entity.hpp

```diff
@@ -241,13 +241,24 @@ class Entity {
         /**
          * @param UID the entity's unique identifier to be set
          */
+
+        ENGINE_API void SetDirection(glm::vec3 rot);
```

Comment:
This has been placed between the doxygen comment of `SetUniqueIdentifier` and that method. This should be relocated and have its own documentation as well.